draft-ietf-teep-protocol.md

```diff
@@ -1512,6 +1500,26 @@ Compromised Time Source
   source of time, such as {{RFC8915}}.  A compromised time source could
   thus be used to subvert such validity checks.
 
+# Privacy Considerations {#privacy}
+
+Depending on
+the properties of the attestation mechanism, it is possible to
+uniquely identify a device based on information in the
+attestation payload or in the certificate used to sign the
+attestation payload.  This uniqueness may raise privacy concerns. To lower the
+privacy implications the TEEP Agent MUST present its
+attestation payload only to an authenticated and authorized TAM and when using
+EATS, it SHOULD use encryption as discussed in {{I-D.ietf-rats-eat}}, since
```

Comment:
Should it be EATs ? or maybe "When using EAT, ..."

Reply:
Does TEEP not use HTTPS?
If it doesn't, then how can the TAM be authenticated and authorized?
If it does, then how the link not encrypted?

Reply:
> Should it be EATs ? or maybe "When using EAT, ..."

Yes it should be "EATs" (i.e., tokens, plural)

> Does TEEP not use HTTPS?

It recommends but does not strictly require HTTPS.   As shown in Figure 5 in https://www.ietf.org/archive/id/draft-ietf-teep-architecture-18.html#name-teep-broker-implementation- the TLS session may or may not be considered trusted (in models A and B it is not, since the HTTP implementation is outside the TEEP).

> If it doesn't, then how can the TAM be authenticated and authorized?

TEEP uses COSE for end-to-end authentication and authorization at the TEEP protocol layer.

Reply:
Updated